src/components/ebay-carousel/component.js

```diff
@@ -92,10 +92,13 @@ function onRender() {
         // Ensure only visible items within the carousel are focusable.
         // We don't have access to these items in the template so me must update manually.
         forEls(listEl, itemEl => {
-            focusables(itemEl).forEach(itemEl.getAttribute('aria-hidden') !== 'true'
-                ? child => child.removeAttribute('tabindex')
-                : child => child.setAttribute('tabindex', '-1')
-            );
+            cleanupFocusables.call(this);
+            this.cancelFocusList.push(focusables(itemEl, false, (focusEl) => {
```

Comment:
Hmm... This is a bit weird. I'm almost thinking deferring this to `makeup-focusables` might not have been the right call 😓. We shouldn't need to create a RAF for each item...

Reply:
Possibly. I think a cleaner way would be to allow focusables to pass in an array and then return only the focused elements for each or something.